tests/Dockerfile

```diff
@@ -21,7 +21,7 @@ FROM iot-lab-gateway
 
 # Update
 RUN apt-get update && \
-    apt-get install -y python-pip gdb
+    apt-get install -y python-pip valgrind gdb
```

Comment:
This looks unrelated

Reply:
This is because we're doing the coverage for the control_node_serial as well, and valgrind is a dependency for that

Reply:
I could separate in 2 PRs (one to test control_node_serial, one for codecov), is that necessary ?

Reply:
> I've reworked the code (mainly ignoring the coverage inside control_node_serial/tests/utils)

You are saying 2 contradictory things. I'm a bit lost.

No need for 2 PRs, just good reasons

Reply:
before the code I added yesterday, we had coverage in codecov.io just fine, but the coverage for control_node_serial was artifically low because it was not ignoring the control_node_serial/tests/utils folder (which contains some binary files). Sorry if I'm not clear ^^

Reply:
That's fine, I understand now.